blocks/card-carousel/card-carousel.js

```diff
@@ -0,0 +1,172 @@
+  span.classList.add('active-nav');
+
+  // Compute the gap to add to the location
+  let gap = 0;
+  if (precedingCard) {
+    const prevRect = precedingCard.getBoundingClientRect();
+    gap = rect.x - (prevRect.x + prevRect.width);
+  }
+
+  slides.scrollTo(idx * (rect.width + gap), slides.scrollHeight);
+}
+
+export function scrollToAdjacent(spans, slideDivs, slides, next, doc) {
+  const curActive = spans.findIndex((s) => s.classList.contains('active-nav'));
+  if (curActive === -1) {
+    return;
+  }
+
+  // compute the next active element, wrapping around on over- or underflow.
+  const newActive = (curActive + (next ? 1 : -1) + spans.length) % spans.length;
+  if (slideDivs.length <= newActive) {
+    return;
+  }
+
+  scrollToCard(
+    newActive,
+    slideDivs[newActive],
+    newActive > 0 ? slideDivs[newActive - 1] : null,
+    slides,
+    spans[newActive],
+    doc,
+  );
+}
+
+let MAX_BUTTONS = 10;
+export default async function decorate(block) {
+  const lang = getLanguage();
+  const placeholders = await fetchPlaceholders(lang);
+
+  const idxPrefix = lang === 'en' ? '' : `${lang}-`;
+  const json = await fetchIndex('query-index', `${idxPrefix}career-testimonials`);
+  const data = shuffleArray(filterIncompleteEntries(json));
+
+  const careerSlider = document.createElement('div');
+  careerSlider.classList.add('career-slider');
+
+  const careerSlides = document.createElement('div');
+  careerSlides.classList.add('career-slides');
+
+  const slideDivs = [];
+  if (block.children && block.children.length > 0) {
+    MAX_BUTTONS = block.children.length - 2;
+    [...block.children].forEach((row) => {
+      const div = document.createElement('div');
+      div.classList.add('career-card');
+
+      const a = document.createElement('a');
+      a.href = '#'; // Todo populate the Link of the stories
+      div.append(a);
+      a.innerHTML = row.innerHTML;
+
+      careerSlides.append(div);
+      slideDivs.push(div);
+    });
+  } else {
+    for (let i = 0; i < data.length; i += 1) {
+      const div = document.createElement('div');
+      div.classList.add('career-card');
+      const a = document.createElement('a');
+      a.href = data[i].path;
+      const pic = createOptimizedPicture(data[i].image, data[i].pagename);
+      a.append(pic);
+
+      const bqc = document.createElement('div');
+      bqc.classList.add('career-card-bqc');
+      const bq = document.createElement('blockquote');
+      bq.textContent = data[i]['career-quote'];
+      bqc.append(bq);
+      a.append(bqc);
+
+      const nm = document.createElement('h6');
+      nm.textContent = data[i].pagename;
+      a.append(nm);
+
+      const role = document.createElement('p');
+      role.textContent = data[i]['career-jobtitle'];
+      a.append(role);
+
+      const link = document.createElement('button');
+      link.textContent = placeholders['career-carousel-readmore'];
+      const arrow = document.createElement('img');
```

Comment:
Nit: Could we instead add this icon from css using an `::after` pseudoelement? I feel it would be cleaner that way